h5grove/encoders.py

```diff
@@ -77,6 +78,12 @@ def npy_stream(array: Sequence[Number]) -> Generator[bytes, None, None]:
         yield chunk.tobytes("C")
 
 
+def convert_tiff(data: np.ndarray):
+    with io.BytesIO() as buffer:
+        tifffile.imwrite(buffer, data, photometric="minisblack")
```

Comment:
Did you checked what happens with special cases: strings/scalar, not 2D array, complex numbers?

Reply:
No but the same way `npy` is not meant be used with such cases, I don't expect the function to be called or to handle them.

If such a thing happens, I am happy to let the code raise an error by itself

Reply:
OK, sounds good.